api/src/main/java/org/apache/cloudstack/api/command/user/vm/ResetVMSSHKeyCmd.java

```diff
@@ -78,8 +84,11 @@
     /////////////////// Accessors ///////////////////////
     /////////////////////////////////////////////////////
 
-    public String getName() {
-        return name;
+    public List<String> getNames() {
+        List<String> keypairnames = new ArrayList<String>();
+        if(names != null) { keypairnames = names; }
+        if(name != null && !name.isEmpty()) { keypairnames.add(name); }
```

Comment:
According to our [Coding conventions](https://cwiki.apache.org/confluence/display/CLOUDSTACK/Coding+conventions), section `Files, Layout and whitespace`, item 7, `if` clauses should be:

```java
if (condition) {
    statements;
}
```